### PR TITLE
Added an audit database

### PR DESCRIPTION
**Pull request by MarshallAsch**
Closes: #136 

Added an audit database to keep an audit log of important events and who ran them. 
---

Checklist:

- [x] Add automated tests cases
- [x] Run `npm run lint-fix` and ensure linter is still passing
- [x] Verify that all automated tests pass
- [x] Verify that the changes work as expected on Discord
- [ ] Update the README and other applicable documentation pages
- [ ] Update the changelog

### CODE REVIEW

File: models/audit/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const audit = require('./audit.js');
+
+// export classes
+module.exports = {
+    Adudit: audit.Adudit,
```
Comment (SoorajModi) [inline models/audit/index.js]: I think you misspelt this?

Comment (MarshallAsch) [reply models/audit/index.js]: indeed I did, although I sweat I fixed it.
and it says that your comment is outdated.

Comment (MarshallAsch) [reply models/audit/index.js]: ya this was fixed I think you reviewed an old version of the code

**Review by SoorajModi — APPROVED** [review]

File: globalConfig.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"mongoUri":"mongodb://127.0.0.1:57201/7b1651a9-dd9c-4489-949f-b5b37f7dbc71?"}
```
Comment (SoorajModi) [inline globalConfig.json:1]: We should have a newline, but that can be added later

Comment (MarshallAsch) [reply globalConfig.json:1]: this is supposed to be git ignored..... I had thought I removed it

**Pull request merged**